pkg/cmd/helm/release/release_test.go

```diff
@@ -113,3 +113,50 @@ func TestStepHelmReleaseWithArtifactory(t *testing.T) {
 		t.Logf("ran: %s\n", c.CLI())
 	}
 }
+
+func TestStepHelmReleaseWithNexus(t *testing.T) {
```

Comment:
could we change this to table driven tests? All the 3 test cases are very similar.